docker.md

```diff
@@ -21,9 +21,26 @@ Start the docker containers with the following command. Make sure to execute thi
 $ make up
 ```
 
+**Warning:** If you also have the `udb3-backend` container running, you will get errors because of colliding port numbers for MySQL and several other services.
+You can fix this with adding to the `.env` file:
+```
+FORWARD_PHP_PORT=8001
```

Comment:
I would put this part in the appconfig file with the default ports. And include here just a note that a developer can change those on his personal machine.